### DISCUSSION TITLE
Do we have an implement of GraphMAE?

### DISCUSSION THREAD

**Discussion by finquick** [pyg-team/pytorch_geometric#5870] [General]
There is a new work of pretrain Graph model, that called: GraphMAE: Self-Supervised Masked Graph Autoencoders is implement by DGL, do we have an implement of this paper use PYG? Thanks!

**Comment by rusty1s** [comment]
We don't have an example for this (yet). If you want, we can work on this together :)

**Comment by MH-limarco** [comment]
I'm sorry to go back to such an old question.
But I found the official version of graphMAE-pyG
https://github.com/THUDM/GraphMAE/tree/pyg

I think this might help with this problem, and if needed as a new example to help organize the code, I think I can help, as I recently researched the content :)
